src/commands/rsync.ts

```diff
@@ -35,9 +40,24 @@ export const rsyncCommand = async () => {
         return;
     }
 
-    try {
-        execSync(`pnpm jetpack rsync ${plugin} ${wpPath}`, {cwd: jetpackRoot});
-    } catch (error) {
-        console.log(error);
-    }
-};
+	// If wpPath includes "jurassic.ninja" we need to open a terminal so the password can be entered.
+	// Otherwise, we can just run the command in the background.
+	if (wpPath.includes('jurassic.ninja')) {
```

Comment:
Hmm, can we not use `terminal` for both? The only difference being `terminal.show();` is only called if it's a `jurassic.ninja` site.

I think the main advantage of executing all these more complex tasks (other than just getting a list of folders/options for example) in a VSCode terminal is you might be able to inspect the logs if something goes wrong. In which case it might be another good idea to add a `terminal.show();` to the `catch` block.